src/chi/components/list-group/list-group.scss

```diff
@@ -0,0 +1,22 @@
+@import 'variables';
+@import 'mixins';
+
+$border: 0.0625rem;
+
```

Comment:
@mattnickles @davidojedacl what about move this class to the _input-wrapper.scss as other classes for grouping form elements are in this scss? And we remove this folder and the file.

Reply:
I have placed .m-listGroup inside _input-wrapper.scss